scripts/release/monitor/requirements.txt

```diff
@@ -0,0 +1,47 @@
+#
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
+#
+#    pip-compile
+#
+annotated-types==0.5.0
```

Comment:
Thought: for tooling do we really need to freeze? 
PROS: reproducible
CONS: maintenance overhead